src/Raven.Server/Monitoring/OpenTelemetry/Constants.cs

```diff
@@ -0,0 +1,200 @@
+        public const string ServerCertificateExpiration = Prefix + "server.certificate.expiration";
+        public const string ServerCertificateExpirationLeft = Prefix + "server.certificate.expiration.left";
+        public const string WellKnownAdminCertificates = Prefix + "well.known.admin.certificates";
+        public const string WellKnownAdminIssuers = Prefix + "well.known.admin.issuers";
+        public const string CertificateExpiringCount = Prefix + "certificate.expiring.count";
+        public const string CertificateExpiredCount = Prefix + "certificate.expired.count";
+        public const string MachineProcessorCount = Prefix + "machine.processor.count";
+        public const string MachineAssignedProcessorCount = Prefix + "machine.assigned.processor.count";
+        public const string ServerBackupsCurrent = Prefix + "server.backups.current";
+        public const string ServerBackupsMax = Prefix + "server.backups.max";
+        public const string ThreadPoolAvailableWorkerThreads = Prefix + "thread.pool.available.worker.threads";
+        public const string ThreadPoolAvailableCompletionPortThreads = Prefix + "thread.pool.available.completion.port.threads";
+        public const string TcpActiveConnections = Prefix + "tcp.active.connections";
+        public const string FeatureAnyExperimental = Prefix + "feature.any.experimental";
+        
+        public class GC
+        {
+            private const string Prefix = ServerWide.Prefix + "gc.";
+            public const string ServerGcCompacted = Prefix + "compacted";
+            public const string ServerGcConcurrent = Prefix + "concurrent";
+            public const string ServerGcFinalizationPendingCount = Prefix + "finalizationpendingcount";
+            public const string ServerGcFragmented = Prefix + "fragmented";
+            public const string ServerGcGeneration = Prefix + "generation";
+            public const string ServerGcHeapSize = Prefix + "heapsize";
+            public const string ServerGcHighMemoryLoadThreshold = Prefix + "highmemoryloadthreshold";
+            public const string ServerGcIndex = Prefix + "index";
+            public const string ServerGcMemoryLoad = Prefix + "memoryload";
+            public const string ServerGcPauseDurations1 = Prefix + "pausedurations1";
+            public const string ServerGcPauseDurations2 = Prefix + "pausedurations2";
+            public const string ServerGcPauseTimePercentage = Prefix + "timepercentage";
+            public const string ServerGcPinnedObjectsCount = Prefix + "pinnedobjectscount";
+            public const string ServerGcPromoted = Prefix + "promoted";
+            public const string ServerGcTotalAvailableMemory = Prefix + "totalavailablememory";
+            public const string ServerGcTotalCommitted = Prefix + "totalcommited";
+            public const string ServerGcLohSize = Prefix + "gclohsize";
```

Comment:
is gc needed here in the name?

Reply:
Well, I've moved whole GC to meter `ravendb.serverwide.gc`